src/current/v24.3/follower-reads.md

```diff
@@ -175,9 +173,61 @@ This example performs a bounded staleness follower read against a [demo cluster]
     (7 rows)
     ~~~
 
-### Verify that a cluster is performing follower reads
+### Verify that CockroachDB is performing follower reads
+
+To verify that a cluster is performing follower reads, go to the [**Custom Chart** debug page in the DB Console]({% link {{ page.version.version }}/ui-custom-chart-debug-page.md %}) and add the metric `follower_read.success_count` to the time-series graph. The number of follower reads performed by your cluster will be shown.
```

Comment:
The metric name is `follower_reads.success_count` (note `s` after `read`).

Reply:
Fixed!